tests/test_base_scalers.py

```diff
@@ -0,0 +1,114 @@
+from unittest.mock import patch
+import json
+import os
+
+import pytest
+
+from app.base_scalers import BaseScaler, AwsBaseScaler, DbQueryScaler
+
+
+class TestBaseScaler:
+    def test_init_assigns_basic_values(self):
+        input_attrs = {
```

Comment:
id stick this up at the top of the module if you're reusing it everywhere. but be careful if it's ever modified 😱